README.md

```diff
@@ -109,25 +115,50 @@ We provide examples in Swift and Objective-C. Run `pod install` from the Example
 
 MapboxNavigationUI.swift makes it easy for developers to add turn-by-turn navigation to their iOS application.
 
-|![](https://cloud.githubusercontent.com/assets/764476/22738709/c23dc02e-ee08-11e6-98ff-e003a06dbe87.png) | ![](https://cloud.githubusercontent.com/assets/764476/22749696/8937b2fa-ee2e-11e6-8bf6-6ea593269b9e.png) |
+|![](https://cloud.githubusercontent.com/assets/764476/23636459/567771d2-028a-11e7-95cf-a8832792c67a.png) | ![](https://cloud.githubusercontent.com/assets/764476/23671279/883c63ae-031f-11e7-8396-b404d18881e1.png) |
 | --- | --- |
 
 ### Examples
-We provide examples in Swift and Objective-C. Run `carthage update --platform ios` from the root folder and open `Example/Example.xcworkspace` to try it out.
+We provide examples in Swift and Objective-C. Run `carthage update --platform ios` from the root folder and open `MapboxNavigation.xcodeproj` to try it out.
 
-**Instantiating a navigation UI (RouteViewController)**
+**Set up navigation UI in code**
```

Comment:
Let's make these lines headings instead of boldfacing them. That way we can link to them if necessary.